src/modules/link.h

```diff
@@ -2,10 +2,73 @@
 #ifndef __LINK_MODULE_H__
 #define __LINK_MODULE_H__
 
+#include <stdbool.h>
+#include <Arduino.h>
 #include <stdint.h>
 
+#define ALARM_ECU_POWER_LOSS              0x01
```

Comment:
need to see that this is up to date with latest protocol document. This is up to date with serialpacketv0.26